admin/class-doppler-for-woocommerce-admin.php

```diff
@@ -404,19 +404,19 @@ public function get_checkout_fields() {
 		$last_product = array( "product_names" => array("label" => "Products",
 														"required" => true,
 														"class" => array("form-row-first"),
-														"autocomplete" => "product-name",
+														"autocomplete" => "products_names",
 														"type" => "string",
 														"priority"=>10),
-								"product_total" => array("label" => "Total Ammount",
+								"product_total" => array("label" => "Total ammount",

```

Comment:
Amount es con una sola M.
```suggestion
								"product_total" => array("label" => "Total amount",

```

Reply:
En varios lugares esta el mismo typo.